sdk/keyvault/azure-keyvault-secrets/azure/keyvault/secrets/_client.py

```diff
@@ -86,7 +86,7 @@ def set_secret(self, name, value, **kwargs):
         Keyword arguments
             - **enabled** (bool): Whether the secret is enabled for use.
             - **tags** (dict[str, str]): Application specific metadata in the form of key-value pairs.
-            - **content_type** (str): An arbitrary string indicating the type of the secret, e.g. 'password'
+            - **content_type** (str): A descriptive string indicating the type of the secret, e.g. 'password'
```

Comment:
'Arbitrary' is apt because there are no illegal values. 'An arbitrary string describing'?

Reply:
ok, switched back!

Reply:
Did you push the commit?